src/main/kotlin/Main.kt

```diff
@@ -750,7 +771,12 @@ fun showPreview(
         row {
             button(blankCommand named "Распределено ролей: ${pairs.size}")
         }
-        button(previewCommand named "🔄 Перераздать", game.id, messageId)
+        button(
+            toggleHideRolesModePreviewCommand named
+                    if (hideRolesMode) "👓 Показывать роли" else toggleHideRolesModePreviewCommand.name,
```

Comment:
Would be more readable in my opinion if both sides of this if are plain strings. Copying one string isn't that big of a deal.